src/pages/WorkflowSource.js

```diff
@@ -0,0 +1,52 @@
+import { Fragment, useState } from 'react'
+import { div, h, h2, h3 } from 'react-hyperscript-helpers'
+import { ButtonPrimary, headerBar, IdContainer, makeInlineDockLink } from 'src/components/common'
+import { TextArea } from 'src/components/input'
+import { FormLabel } from 'src/libs/form'
+
+
+export const WorkflowSource = () => {
+  // State
+  const [workflowUrl, setWorkflowUrl] = useState()
+
+  return div([
+    headerBar(),
+    div({ style: { margin: '4rem' } }, [
+      div([
+        h2(['Submit a workflow']),
+        div(['Submit your Terra workflows with the Cromwell engine. Full featured workflow submissions coming soon!'])
+      ]),
+      div({ style: { marginTop: '2rem' } }, [
+        h3(['Provide a URL to a workflow']),
+        div([
+          h(Fragment, [
+            `You can browse for workflows in `,
+            makeInlineDockLink('https://firecloud.dsde-dev.broadinstitute.org/?return=terra#methods', 'Broad Methods Repository', 14),
+            ` or in `,
+            makeInlineDockLink('https://staging.dockstore.org/search?descriptorType=WDL&entryType=workflows&searchMode=files', 'Dockstore', 14),
+            `.`
+          ])
+        ]),
+        h(IdContainer, [id => h(Fragment, [
+          h(FormLabel, { htmlFor: id }),
+          h(TextArea, {
+            id,
+            style: { height: 100 },
+            placeholder: 'Paste workflow URL here',
```

Comment:
Possibly worth noting that only public, unauthenticated URLs ending in `.wdl` work, such as
```
https://raw.githubusercontent.com/openwdl/learn-wdl/master/1_script_examples/1_hello_worlds/3_hello_input_task/input-task.wdl
```
while these do not because they are HTML pages (even though they sometimes end in `.wdl`!)
```
https://firecloud.dsde-dev.broadinstitute.org/?return=terra#methods/anichols-shared-namespace/anichols-private-method/1
https://github.com/Jemimacat/TRUST4_bam_hg38/blob/master/TRUST4_bam_hg38.wdl
https://staging.dockstore.org/workflows/github.com/Jemimacat/TRUST4_bam_hg38:master?tab=info
```

Reply:
Good point. I will add that in the current ticket I am working on. This is because I already have a new branch which is branched out from `ss_accept_workflow_url`, and changing anything in `ss_accept_workflow_url` might lead to some conflicts while rebasing as I already have lot of changes in the new branch.